tools/roslaunch/scripts/roscore

```diff
@@ -56,6 +56,9 @@ def _get_optparse():
     parser.add_option("-t", "--timeout",
                       dest="timeout",
                       help="override the socket connection timeout (in seconds).", metavar="TIMEOUT")
+    parser.add_option("--set-master-logger-level",
```

Comment:
The `set` of the options seems redundant to me. How about just `--master-logger-level`?